include/modem/location.h

```diff
@@ -693,6 +705,21 @@ void location_config_defaults_set(
  */
 const char *location_method_str(enum location_method method);
 
+/**
+ * @brief Get location data details from the location event data.
+ *
+ * @details The :c:struct:`location_data_details` structure is located in a different place in the
```

Comment:
I am pretty sure this notation does not work within doxygen. It only works in the rst docs. I believe you would need to refer to structures with @ref, as mentioned in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/dev_model_and_contributions/documentation/styleguide.html#references

Reply:
Fixed.
Thanks for pointing this out. I noticed I'm mixing these when writing both .h and .rst files at the same time.